### ISSUE TITLE
Connect to github popup and failed :PackerInstall

### ISSUE THREAD

**Issue by luiq54** [wbthomason/packer.nvim#641]
- `nvim --version`:
  NVIM v0.5.1
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
- `git --version`: git version 2.33.0.windows.2
- Operating system/version: Windows 10 May 2021
- Terminal name/version:

### Steps to reproduce and behavior 
 I added `use 'williamboman/nvim-lsp-installer'` in the startup function and then calling :PackerInstall creates a popup asking for GitHub credentials. It asks to paste your token in or login through browsers. Accepting or closing the popups fails :PackerInstall nonetheless

### packer files
<details>
<summary>packer compiled file</summary>
-- Automatically generated packer.nvim plugin loader code

if vim.api.nvim_call_function('has', {'nvim-0.5'}) ~= 1 then
  vim.api.nvim_command('echohl WarningMsg | echom "Invalid Neovim version for packer.nvim! | echohl None"')
  return
end

vim.api.nvim_command('packadd packer.nvim')

local no_errors, error_msg = pcall(function()

  local time
  local profile_info
  local should_profile = false
  if should_profile then
    local hrtime = vim.loop.hrtime
    profile_info = {}
    time = function(chunk, start)
      if start then
        profile_info[chunk] = hrtime()
      else
        profile_info[chunk] = (hrtime() - profile_info[chunk]) / 1e6
      end
    end
  else
    time = function(chunk, start) end
  end
  
local function save_profiles(threshold)
  local sorted_times = {}
  for chunk_name, time_taken in pairs(profile_info) do
    sorted_times[#sorted_times + 1] = {chunk_name, time_taken}
  end
  table.sort(sorted_times, function(a, b) return a[2] > b[2] end)
  local results = {}
  for i, elem in ipairs(sorted_times) do
    if not threshold or threshold and elem[2] > threshold then
      results[i] = elem[1] .. ' took ' .. elem[2] .. 'ms'
    end
  end

  _G._packer = _G._packer or {}
  _G._packer.profile_output = results
end

time([[Luarocks path setup]], true)
local package_path_str = "C:\\Users\\DARSHA~1\\AppData\\Local\\Temp\\nvim\\packer_hererocks\\2.1.0-beta3\\share\\lua\\5.1\\?.lua;C:\\Users\\DARSHA~1\\AppData\\Local\\Temp\\nvim\\packer_hererocks\\2.1.0-beta3\\share\\lua\\5.1\\?\\init.lua;C:\\Users\\DARSHA~1\\AppData\\Local\\Temp\\nvim\\packer_hererocks\\2.1.0-beta3\\lib\\luarocks\\rocks-5.1\\?.lua;C:\\Users\\DARSHA~1\\AppData\\Local\\Temp\\nvim\\packer_hererocks\\2.1.0-beta3\\lib\\luarocks\\rocks-5.1\\?\\init.lua"
local install_cpath_pattern = "C:\\Users\\DARSHA~1\\AppData\\Local\\Temp\\nvim\\packer_hererocks\\2.1.0-beta3\\lib\\lua\\5.1\\?.so"
if not string.find(package.path, package_path_str, 1, true) then
  package.path = package.path .. ';' .. package_path_str
end

if not string.find(package.cpath, install_cpath_pattern, 1, true) then
  package.cpath = package.cpath .. ';' .. install_cpath_pattern
end

time([[Luarocks path setup]], false)
time([[try_loadstring definition]], true)
local function try_loadstring(s, component, name)
  local success, result = pcall(loadstring(s))
  if not success then
    vim.schedule(function()
      vim.api.nvim_notify('packer.nvim: Error running ' .. component .. ' for ' .. name .. ': ' .. result, vim.log.levels.ERROR, {})
    end)
  end
  return result
end

time([[try_loadstring definition]], false)
time([[Defining packer_plugins]], true)
_G.packer_plugins = {
  ["nvim-lsp-config"] = {
    loaded = true,
    path = "C:\\Users\\Darshan Phaldesai\\AppData\\Local\\nvim-data\\site\\pack\\packer\\start\\nvim-lsp-config"
  },
  ["nvim-lsp-installer"] = {
    loaded = true,
    path = "C:\\Users\\Darshan Phaldesai\\AppData\\Local\\nvim-data\\site\\pack\\packer\\start\\nvim-lsp-installer"
  },
  ["nvim-treesitter"] = {
    loaded = true,
    path = "C:\\Users\\Darshan Phaldesai\\AppData\\Local\\nvim-data\\site\\pack\\packer\\start\\nvim-treesitter"
  },
  ["packer.nvim"] = {
    loaded = true,
    path = "C:\\Users\\Darshan Phaldesai\\AppData\\Local\\nvim-data\\site\\pack\\packer\\start\\packer.nvim"
  },
  ["tokyonight.nvim"] = {
    loaded = true,
    path = "C:\\Users\\Darshan Phaldesai\\AppData\\Local\\nvim-data\\site\\pack\\packer\\start\\tokyonight.nvim"
  },
  ["vim-commentary"] = {
    loaded = true,
    path = "C:\\Users\\Darshan Phaldesai\\AppData\\Local\\nvim-data\\site\\pack\\packer\\start\\vim-commentary"
  },
  ["vim-surround"] = {
    loaded = true,
    path = "C:\\Users\\Darshan Phaldesai\\AppData\\Local\\nvim-data\\site\\pack\\packer\\start\\vim-surround"
  }
}

time([[Defining packer_plugins]], false)
if should_profile then save_profiles() end

end)

if not no_errors then
  vim.api.nvim_command('echohl ErrorMsg | echom "Error in packer_compiled: '..error_msg..'" | echom "Please check your config for correctness" | echohl None')
e

</details>

**Comment by luiq54** [comment]
> To clarify, you have successfully installed other plugins without getting the GH login prompt? 

Yes. Interestingly, I  ran :PackerSync today and it error-ed out with GH popup on `lsp-config` instead of `nvim-lsp-installer`. The latter installed just fine.  Only logs I could find were these 

```
[WARN  09/10/2021 11:28:43 3566999932400] ...a\site\pack\packer\start\packer.nvim\lua\packer\jobs.lua:87: Killing git due to timeout!
[WARN  09/10/2021 11:30:27 3671606818699] ...ite\pack\packer\start\packer.nvim\lua\packer\display.lua:607: No plugin selected!
[WARN  09/10/2021 11:30:36 3680484431300] ...ite\pack\packer\start\packer.nvim\lua\packer\display.lua:607: No plugin selected!
```
I got these from C:\Users\\****\AppData\Local\Temp\nvim ( ~~ stdpath('cache'))

**Comment by wbthomason** [comment]
Ah, sorry - usually, when `packer` fails to complete an operation, you can press `<cr>` on the failed operation to expand it and get error output. I'm guessing that didn't work, from the last two lines of your log, but if you haven't tried it yet, could you please?

Are you able to carry out `git` operations (in particular, cloning) manually from the terminal you run Neovim in without getting this prompt?

**Comment by luiq54** [comment]
I was using the error report in the wrong way. Here is the error : 
```
 ✗ Failed to install neovim\nvim-lsp-config
  Errors:
    Cloning into 'C:\Users\darsh\AppData\Local\nvim-data\site\pack\packer\start\nvim-lsp-config'...
    fatal: helper error (-1): authentication prompt was canceled
    fatal: could not read Username for 'https://github.com': terminal prompts disabled

```
Normal git clones work fine. I tried cloning both repos in a random directory.

**Comment by erikkrieg** [comment]
I also hit some confusion with this error message. There was a subtle typo in the repo string but I went debugging in the totally wrong direction.
